docs/src/main/sphinx/release/release-468.md

```diff
@@ -0,0 +1,32 @@
+# Release 468 (11 Dec 2024)
+
+## General
+
+* Add experimental support for Python user-defined functions. ({issue}`24378`)
```

Comment:
If a release note has ever needed a link to docs, this is that release note.

Reply:
I know.. I am frantically working on it ..

Reply:
Its a big PR .. and not yet ready - have to refactor the SQL routine docs completely. I am hoping to get it done asap